packages/Meshfree/test/tstSVD.cpp

```diff
@@ -0,0 +1,157 @@
+/****************************************************************************
+ * Copyright (c) 2012-2018 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Teuchos_UnitTestHarness.hpp>
```

Comment:
@aprokop Did you have more SVD tests?

Reply:
The only test I had (in a separate repo) was along the lines of:
```c++
scalar_type check_svd(int num_matrices, int n, matrices_type::const_type As, matrices_type::const_type pseudoAs) {
  auto As_host       = Kokkos::create_mirror_view(As);
  auto pseudoAs_host = Kokkos::create_mirror_view(pseudoAs);

  Kokkos::deep_copy(As_host,       As);
  Kokkos::deep_copy(pseudoAs_host, pseudoAs);

  scalar_type max_norm = 0.0;
  for (int matrix_id = 0; matrix_id < num_matrices; matrix_id++) {
    auto A       = Kokkos::subview(As_host,       Kokkos::make_pair(matrix_id*n*n, (matrix_id+1)*n*n));
    auto pseudoA = Kokkos::subview(pseudoAs_host, Kokkos::make_pair(matrix_id*n*n, (matrix_id+1)*n*n));

    scalar_type norm = 0.0;
    for (int i = 0; i < n; i++)
      for (int j = 0; j < n; j++) {
        scalar_type t = 0.0;
        for (int k = 0; k < n; k++)
          t += A(i*n+k)*pseudoA(k*n+j);
        norm += (i != j ? t*t : (t-1)*(t-1));
      }
    norm = sqrt(norm);

    if (norm > max_norm)
      max_norm = norm;
  }

  return max_norm;
}
```